manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -151,6 +151,12 @@ spec:
             value: {{ .Values.global.jwtPolicy }}
           - name: PILOT_CERT_PROVIDER
             value: {{ .Values.global.pilotCertProvider }}
+          - name: CA_ADDR
+          {{- if .Values.global.caAddress }}
```

Comment:
can the default be added to profiles/default.yaml instead?

Reply:
Yes, we can basically add any custom flag into profiles/default through .Values.meshConfig.defaultConfig.proxyMetadata. We have the single place to hold the duplicated config.
I would prefer have another PR to do it for JWT_PILOCY and PILOT_CERT_PROVIDER, etc as well.